src/spring/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+1.15.0
```

Comment:
Do we need to also update the version in `setup.py`?

Reply:
No, the PR will be released with other PRs together.